src/auth/dto/authcreds.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsString, IsNotEmpty, IsEmail } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class AuthCredsDto {
+  @IsString()
+  @ApiProperty({ description: 'The email of the user' })
+  @IsEmail()
+  @IsNotEmpty()
+  email: string;
```

Comment:
PrimathonAICodeReviewer

Consider adding a validation decorator for the maximum length of the email field to prevent excessively long inputs.
```diff
+   @MaxLength(255)
```
  


<!-- This is an auto-generated comment by OSS PrimathonAICodeReviewer -->